subscription-engine/kafka-ws/README.md

```diff
@@ -292,30 +292,74 @@ operators](###applying-conditions-where) for more info
-KAFKA_WS_PORT          | Kafka WS port                                                                               | 8080                                | valid port                    |
-KAFKA_WS_TLS           | Enables TLS (Needs: KAFKA_WS_TLS_CA_FILE, KAFKA_WS_TLS_KEY_FILE and KAFKA_WS_TLS_CERT_FILE) | false                               | true or false / 1 or 0        |
-KAFKA_WS_TLS_CA_FILE   | Kafka WS ca file location                                                                   | /opt/kafka-ws/certs/ca-cert.pem     | valid path                    |
-KAFKA_WS_TLS_KEY_FILE  | Kafka WS key file location                                                                  | /opt/kafka-ws/certs/server-key.pem  | valid path                    |
-KAFKA_WS_TLS_CERT_FILE | Kafka WS certificate file location                                                          | /opt/kafka-ws/certs/server-cert.pem | valid path                    |
-KAFKA_WS_JWT_EXP_TIME  | Enables use of expiration time from the informed JWT when requesting a *single-use ticket*. | false                               | true or false / 1 or 0        |
-TICKET_EXPIRATION_SEC  | Duration time (in seconds) of the *single-use ticket*.                                      | 60                                  | seconds                       |
-TICKET_SECRET          | Secret used to sign *single-use tickets* and prevent forgery                                | Random value                        | string                        |
-KAFKA_WS_MAX_LIFE_TIME | Maximum lifetime of a connection   (-1 to disable)                                          | 7200                                | seconds                       |
-REDIS_HOST             | Redis host                                                                                  | kafka-ws-redis                      | string                        |
-REDIS_PORT             | Redis port                                                                                  | 6379                                | number                        |
-REDIS_DATABASE         | Redis database                                                                              | 1                                   | number                        |
-KAFKA_WS_REQUEST_CERT  | Whether the service should request for a client certificate or not                          | false                               | boolean                       |
-
-__NOTE THAT__ a websocket connection is closed by the server when certain conditions are met. If KAFKA_WS_JWT_EXP_TIME is set to true, the server will consider this value for closing the connection if it is greater than the KAFKA_WS_MAX_LIFE_TIME. If KAFKA_WS_JWT_EXP_TIME is set to false and KAFKA_WS_MAX_LIFE_TIME is set to -1, the server will never close a connection by its duration.
+You can select the configuration file via the `KAFKA_WS_APP_USER_CONFIG_FILE` variable. Its default value is `production.env`
```

Comment:
I see that you got this part from the V2K README. I updated it the other day adding this:
```
Check the [config directory](./config) for the user configurations that are
available by default.
```

I think you could add it too.

Reply:
Great!